Documentation/internals/cilium_operator.rst

```diff
@@ -50,6 +50,14 @@ Cilium. The following custom resources are registered by the Cilium Operator:
 -  CiliumExternalWorkload
 -  CiliumIdentity
 -  CiliumLocalRedirectPolicy
+-  CiliumEgressGatewayPolicy
+-  CiliumEndpointSlice
+-  CiliumClusterwideEnvoyConfig
+-  CiliumClusterwideEnvoyConfig
+-  CiliumBGPPeeringPolicy
+-  CiliumLoadBalancerIPPool
+-  CiliumNodeConfig
+-  CiliumCIDRGroup
```

Comment:
BTW, can't we say, "Operator registers all Cilium-related CRDs" instead of listing all CRDs here? So that we don't have to take care of updating this list. Otherwise, should we automate the update?

Reply:
I was having the same reaction, I was about to ask if there was any kind of test that we should add to make sure we don't miss any in the future.

Would be nice to have the update automated I think. For example we could:

- Move this to a dedicated RST fragment, that we include from `cilium_operator.rst`
- Generate this fragment from a dedicated `Documentation/Makefile` target

Reply:
Automation would be nice, and should be a pretty trivial change to submit along the lines of @qmonnet's suggestion above. That's a fairly common pattern. PR https://github.com/cilium/cilium/pull/24968 is doing something a bit like this, but tracking CRDs wouldn't need to be quite so elaborate.

Reply:
Sounds like a plan. I created a new issue for this and assigned myself (https://github.com/cilium/cilium/issues/25750).